<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="PasswdInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastNameInput">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>140</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="RegisterText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;REGISTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LastNameText">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FirstNameInput">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>140</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmPasswdInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegistrBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgba(255, 255, 255, 100);
border-radius: 10px;
color: rgba(192, 192, 192, 240);
color: black;
text-align: center;
font: 700 11pt &quot;Segoe Print&quot;;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(0, 0, 0, 240);
text-align: center;
font: 700 11pt &quot;Segoe Print&quot;;
}
QPushButton:pressed{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(128, 128, 128, 240);
text-align: center;
font: 700 11pt &quot;Segoe Print&quot;;
}</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLabel" name="FirstNameText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="BlurImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(192, 192, 192, 150);
border-radius: 30px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PasswdText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmailText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmPasswdText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Backgroung">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/darkBg.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UserText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorText">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>484</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>Backgroung</zorder>
  <zorder>BlurImage</zorder>
  <zorder>PasswdInput</zorder>
  <zorder>LastNameInput</zorder>
  <zorder>EmailInput</zorder>
  <zorder>RegisterText</zorder>
  <zorder>LastNameText</zorder>
  <zorder>FirstNameInput</zorder>
  <zorder>ConfirmPasswdInput</zorder>
  <zorder>RegistrBtn</zorder>
  <zorder>FirstNameText</zorder>
  <zorder>PasswdText</zorder>
  <zorder>EmailText</zorder>
  <zorder>ConfirmPasswdText</zorder>
  <zorder>UserInput</zorder>
  <zorder>UserText</zorder>
  <zorder>ErrorText</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
